web/docs/setup-vast/tune.md

```diff
@@ -29,17 +29,18 @@ to the scheduler. As a result, other actors scheduled on the same thread may
 have to wait a little longer.
 
 The option `vast.import.batch-size` sets an upper bound for the number of events
-per table slice.
+per table slice. It defaults to 64k.
```

Comment:
It defaults to 64 Ki, actually.

Reply:
I will use 65536 instead to avoid any kind of confusion for a reader.

Reply:
Should we also mention that this number is the default and recommended batch size for the Apache Feather V2 file format, just to give some sense to the seemingly random number?

Reply:
Let's do that in a follow-up PR if desired.